src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/TypedContentTypeBuilder.cs

```diff
@@ -47,19 +50,19 @@ public void Build(FieldType contentQuery, ContentTypeDefinition contentTypeDefin
                     continue;
                 }
 
-                var activator = typeActivator.GetTypeActivator(part.PartDefinition.Name);
-
-                var queryGraphType = typeof(ObjectGraphType<>).MakeGenericType(activator.Type);
+                var partType = PartTypes.GetOrAdd(part.PartDefinition.Name, key => typeActivator.GetTypeActivator(key).Type);
```

Comment:
Now you are caching this one. So if the other one is not cache I assume it's because it doesn't have to?